docs/stacks-guide-azure.md

```diff
@@ -48,24 +49,27 @@ document][crossplane-concepts] for that.
 
 ## Install the Azure Stack
 
-After Crossplane has been installed, it can be extended with more functionality
-by installing a Crossplane Stack! Let's install the stack for Microsoft Azure to
-add support for that cloud provider. We can use the Crossplane CLI for this
-operation:
+After Crossplane has been installed, it can be extended with more
+functionality by installing a [Crossplane Stack][stack-docs]! Let's
+install the [stack for Microsoft Azure][stack-azure] to add
+support for that cloud provider. We can use the [Crossplane
+CLI][crossplane-cli] for this operation. Since this is an infrastructure
+stack, we need to specify that it's cluster-scoped by passing the
+`--cluster` flag.
 
-```
-kubectl crossplane stack install 'crossplane/stack-azure:master' stack-azure
-```
-To install to a particular namespace, you can use the `generate-install` command
-and pipe it to `kubectl apply` instead, which gives you more control over how
-the stack's installation is handled. Everything is Kubernetes object!
+To install to a specific namespace, we can use the `generate-install`
+command and pipe it to `kubectl apply` instead, which gives us more
+control over how the stack's installation is handled. Everything is
+a Kubernetes object!
 
-Since this is an infrastructure stack, we need to specify that it's
-cluster-scoped via `--cluster` flag.
 ```
 kubectl crossplane stack generate-install --cluster 'crossplane/stack-azure:master' stack-azure | kubectl apply --namespace crossplane-system -f -
 ```
 
+If we wanted to use whatever the current namespace is, we could have
+used `kubectl crossplane stack install` instead of using
+`generate-install`.
+
 We have installed the Azure stack into the `crossplane-system` namespace, but we
 want to group our Azure-specific resources in their own environment namespaces.
 For the purpose of this guide, we will create all Azure-specific resources in
```

Comment:
I believe we could remove the part about `app-project1-dev` here since it is in the general stacks guide. Not vital that it is removed though